File: Python/Sample Paper - 03/Armstrong.py
n = int(input())
l = len(str(n))
s = n
d = 0
while(n>0):
    i = n%10
    d += i**l
    n = n//10
if s == d:
    print("Armstrong")
else:
    print("Not Armstrong")
    
    

File: c course/c questons/armstrong.c
#include<stdio.h>
int main()
{
    int n,r,c,sum=0,orig;
    printf("enter the number:");
    scanf("%d",&n);
    orig=n;
    while(n>0)
    {
        r=n%10;
        c=r*r*r;
        sum=sum+c;
        n=n/10;
    }
    if(sum==orig)
    {
        printf("number is armstrong");
    }
    else
    {
        printf("number is not armstrong");
    }
    return 0;
}